test/twofishes/result_test.rb

```diff
@@ -37,7 +37,7 @@
       assert_equal 'Zürich', result.name
     end
 
-    it "should return display_name" do
+    it "should return display_name (rubyfied)" do
```

Comment:
i don't understand what the rubyfied is here for? did you want to separate tests for `result.displayName`and `display_name` instead?